domain/BUILD.bazel

```diff
@@ -22,6 +22,7 @@ kt_android_library(
         ":dagger",
         "//data:persistent_cache_store",
         "//model:onboarding_java_proto_lite",
+        "//model:topic_java_proto_lite",
```

Comment:
This is surprising to me. Do you have an idea on how the library declared this dependency previously?

Reply:
Yes there was no model dependency before

Reply:
This might have been pulled in indirectly through a downstrream dep before.